src/components/personalia/adresser.tsx

```diff
@@ -238,15 +237,25 @@ type Props = Pick<PersonaliaV2Info, 'bostedsadresse'> &
 
 function Adresser(props: Props) {
     const { bostedsadresse, oppholdsadresse, kontaktadresser } = props;
-    const kontaktadresseList = isNotEmptyArray(kontaktadresser)
-        ? kontaktadresser.map((kontaktadresse, index) => <KontaktAdresse kontaktadresse={kontaktadresse} key={index} />)
-        : EMDASH;
+
     return (
-        <Informasjonsbolk header="Adresser" headerTypo="ingress">
-            <BostedsAdresse bostedsadresse={bostedsadresse} />
-            <OppholdsAdresse oppholdsadresse={oppholdsadresse} />
-            {kontaktadresseList}
-        </Informasjonsbolk>
+        <div>
```

Comment:
må dette være en div eller kan det bare være en <> ? :)

Dersom man setter en div kun for å lage en samling av komponenter så kan det være fint å bruke <> eller React.Fragment som det egentlig er isteden. Da vil det ikke bli noe ekstra i DOM'en som i det lange løp (og i større applkasjoner) kan gjøre at komponenter rendrer kjappere og bruker mindre minne.